src/main/java/seedu/duke/data/notebook/TagManager.java

```diff
@@ -11,13 +11,13 @@
  */
 public class TagManager {
 
-    private Map<Tag, ArrayList<Note>> tagMap;
+    private static Map<Tag, ArrayList<Note>> tagMap;
```

Comment:
Don't have to make it static, each Command class have a TagManager object, can access the tagManager from there